### DISCUSSION TITLE
Plotting the miller indices on the IPF

### DISCUSSION THREAD

**Discussion by Yasmine-G** [mtex-toolbox/mtex#1393] [Ask Anything] [answered]
Hi,

I wanted to ask how is it possible to know each grain's miller indices on an EBSD map? 
Also how can we show all the miller indices on the IPF?

Thanks for your time.

Yasmine

**Comment by kilir** [comment, marked as answer]
Hi,
in the broadest sense `inv(orientation)  .* referencedirection` where the orientation would be somethign like `grains(phase).meanOrientation` and `referencedirection` your desired reference direction of type `vector3d`. See here: https://mtex-toolbox.github.io/Documentation.html
For color coding, see here https://mtex-toolbox.github.io/EBSDIPFMap.html

Hope this helps.
Rüdiger
